### DISCUSSION TITLE
Longer slicing time?

### DISCUSSION THREAD

**Discussion by Kezat231** [supermerill/SuperSlicer#1985] [Q&A] [answered]
Today I changed to 2.3.57.6 from 2.3.56.8, Win64.
To slice a simple Benchy my slicing time is now 20 seconds or so instead of the 3 it was before. Is this increased slicing time expected and if not can anything be done on my end to speed it up again? Let me know what information would be helpful to troubleshoot this if it's abnormal.

Also a bonus question if anyone knows. In build 2.3.56.0 the setting "Setting to remove unwanted bottom/top fill on slopes" was added but for the life of me I can't find this new setting. Does anyone know where this is?

**Comment by neophyl** [comment]
Don't know about your slicing time.  The second one though, Print Settings>Perimeters & Shell>Horizontal Shells>No solid Infill over:

**Reply by supermerill** [reply]
yes, just below top/bottom layer count settings.

**Reply by Kezat231** [reply]
Thanks Neophyl and supermerill, I skipped right over that as I was expecting a check box not a entry field for a number. :)

**Comment by supermerill** [comment, marked as answer]
For the slowdown, can you send me the project file from 2.3.56.8 and the one from 2.3.57.6? i'll see what's causing this.

**Reply by Kezat231** [reply]
I have included four versions of the project files 46.8 57.4 57.5 and 57.6. Strangely 2.3.57.4 took 225 seconds, might be useful as a data point?
[increased slice time project files.zip](https://github.com/supermerill/SuperSlicer/files/7638052/increased.slice.time.project.files.zip)

**Reply by supermerill** [reply]
That's due to the new bridge detection. It's doing many more tests to find the right spot. I tried to optimized it a bit, I'll see if I can go further. If I can't get to an acceptable timeframe, I'll add a setting for "quick& dirty" bridge angle compute.

**Reply by supermerill** [reply]
I succeed to reduce the call number from 24k to 3, much better. Will be in next release.

**Reply by Kezat231** [reply]
Fantastic! I'm looking forward to it. Thank you for all that you do on SuperSlicer.

**Reply by Kezat231** [reply]
Can confirm 2.3.57.7 has the slice time back to a fast 3 seconds. Thank you Merill.
